### ISSUE TITLE
Libvirt-bin has trouble starting if -d is stated in /etc/default/libvirt-bin (libvirtd_opts)

### ISSUE THREAD

**Issue by falcon78921** [apache/cloudstack#2546]
##### ISSUE TYPE

 * Other

##### COMPONENT NAME

~~~
Apache CloudStack Agent (4.9.3) & libvirt-bin (1.3.1)
~~~

##### CLOUDSTACK VERSION

~~~
Apache CloudStack 4.9.3
~~~

##### CONFIGURATION

~~~
Basic network zone. Three compute nodes running KVM & libvirt-bin. Ceph is being utilized for Primary Storage.
~~~

##### OS / ENVIRONMENT

~~~
Ubuntu 16.04.4 LTS
~~~

##### SUMMARY

~~~
* Libvirt-bin has trouble starting if -d is stated in /etc/default/libvirt-bin. Referencing this from Apache CloudStack documentation (http://docs.cloudstack.apache.org/projects/cloudstack-installation/en/4.9/hypervisor/kvm.html#install-and-configure-libvirt). 
~~~

##### STEPS TO REPRODUCE

~~~
1.) Add -d in libvirtd_opts="" within /etc/default/libvirt-bin. -l works fine for listening on TCP.
~~~

##### EXPECTED RESULTS

~~~
Libvirt-bin should've started with no errors. The parameters -d -l as libvirtd_opts should exist together, per the CloudStack 4.9 documentation.
~~~

##### ACTUAL RESULTS

~~~
Libvirt-bin fails to start. This also causes the ACS agent to fail as well. If you take out the -d in libvirtd_opts and leave the -l, libvirt-bin works great. I followed this solution to solve the problem:
https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1594902
https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1594902/comments/10
~~~
I thought I would report this, just in case anyone running Ubuntu 16.04 with ACS 4.9.3 has the same problem.

**Comment by rhtyd** [comment]
Thanks @falcon78921 I've found and fixed this in one of the PRs. The issue is caused because daemon-management is done by systemd, and running libvirtd with -d -l causes two copies of libvirtd process to run.

**Comment by falcon78921** [comment]
Thanks @rhtyd I'm happy to help! :)

**Comment by rhtyd** [comment]
Fixed in #2550
